src/inference/tests/unit/cpu_map_parser.cpp

```diff
@@ -1299,6 +1299,418 @@ INSTANTIATE_TEST_SUITE_P(CPUMap,
                                          freq_1sockets_8cores_1_hyperthreading,
                                          freq_1sockets_4cores));
 
+struct LinuxNumactlTestCase {
+    std::vector<int> phy_core_list;
+    std::vector<std::vector<int>> input_proc_type_table;
+    std::vector<std::vector<int>> input_cpu_mapping_table;
+    int _sockets;
+    int _cores;
+    std::vector<std::vector<int>> _proc_type_table;
+    std::vector<std::vector<int>> _cpu_mapping_table;
+};
+
+class LinuxCpuMapNumactrlTests : public CommonTestUtils::TestsCommon,
```

Comment:
minor: your are mixing `numactl` and `numactrl` naming. I would advise to align with the tool name in order to avoid confusion.

Reply:
updated